static/ca.js

```diff
@@ -84,12 +89,20 @@ function parsingDone(results, file) {
   });
 
   countySel = document.getElementById('county-select');
-  populateSelect(countySel, counties);
+  populateSelect(countySel, counties, selectedCounties);
 
   locationTypeSel = document.getElementById('location-type-select');
-  populateSelect(locationTypeSel, locationTypes);
+  populateSelect(locationTypeSel, locationTypes, selectedVenues);
+
+  redoFilter();
 
-  _.each([countySel, locationTypeSel], function(sel) { sel.addEventListener('change', redoFilter); });
+  countySel.addEventListener('change', function() {
+    window.location = "/counties/" + encodeURIComponent(countySel.value);
```

Comment:
Just discovered that there is a bug with how "Chart/All" currently works and how this URL changing scheme works.

When we go to "Chart/All" the URL is like this: http://127.0.0.1:8080/bydate.html?datafilename=raw
When I then pick Beaches from the venue dropdown, this code changes the URL to http://127.0.0.1:8080/bydatesel/ALL/Beaches
but in order to stay in the "Chart/All" space the URL needs to be http://127.0.0.1:8080/bydatesel/ALL/Beaches?datafilename=raw
so that it works with the `raw.csv` file instead of the `grouped.csv` file.

The ?datafilename=raw part is getting dropped and lost.

Reply:
I'm resolving this in a merge i'm doing of something else

Reply:
Sorry, missed that!  I can fix in the am, if you prefer.

Reply:
No worries I fixed it